README.md

```diff
@@ -169,3 +171,55 @@ After the debug is complete, one has to manually clean up the setup.
 
 The following test options are uncommon and meant to use under rare situations:
 + `--debug-execute`: debug command execution over the ssh session
+
+## Switch settings
+
+If a layer 2 switch is used to connect the traffic generator and the DUT, the switch ports facing the servers should have the same native vlan. The native vlan may have different terminologies for different switch brands. For example for dell switch, it is defined as an access vlan on the trunking ports. The vlan tag used for the vlan testing, or the outer vlan used for QinQ testing, has a default value 10. To allow the vlan-tagged traffic through the switch, the switch ports should be set to trunk mode and allow this vlan. In addition, the switch port MTU should be spcified in the `tests/config.yaml`. Here is an port configure example for a dell switch,
+```
+interface ethernet1/1/11:3
+ description "trafficgen port 1"
+ no shutdown
+ switchport mode trunk
+ switchport access vlan 100
+ switchport trunk allowed vlan 10
+ mtu 9216
+ flowcontrol receive off
+ spanning-tree disable
```

Comment:
Do we need to disable spanning tree?

Reply:
We should keep the stp running otherwise the script won't be able to detect the switch existence